_posts/2016-06-30-Rails-5-0-final.markdown

```diff
@@ -0,0 +1,83 @@
+
+There's still more work to be done on this feature, but we're off to a great start. By default, API mode just relies on `#to_json` calls on model classes. But you can either use [Jbuilder][jbuilder], [Active Model Serializers][ams], or look at the new [JSONAPI::Resources][jsonapi] project for a more advanced solution.
+
+Thanks in particular to [Santiago Pastorino][santiagopastorino] and [Jorge Bejar][jorgebejar] for making this happen.
+
+## Other highlights
+
+* One Rails Command instead of the split-brain setup between rake and rails, so now it's `bin/rails db:migrate` instead of `bin/rake db:migrate` (herding by [Kasper Timm Hansen][kasper]).
+* [New Attributes API][attributes-api] by [Sean Griffin][sean].
+* The test runner now reports failures inline, so you don't have to complete the suite to see what went wrong.
+* ApplicationRecord has been born as a default parent class of all models created by the generators.
+* ActiveRecord::Relation#in_batches makes it much easier to deal with record work in batches at a time to lessen memory overloads.
+* Post.where('id = 1').or(Post.where('id = 2')) gives you exactly what you'd think!
+* No more accidentally halting Active Record callbacks because the last statement is false. Now you throw(:abort) explicitly!
+
+You should really checkout the CHANGELOGs, though. There's just so much new and good stuff available in all the frameworks:
+
+* [Action Mailer CHANGELOG](https://github.com/rails/rails/blob/v5.0.0/actionmailer/CHANGELOG.md)
+* [Action Pack CHANGELOG](https://github.com/rails/rails/blob/v5.0.0/actionpack/CHANGELOG.md)
+* [Action View CHANGELOG](https://github.com/rails/rails/blob/v5.0.0/actionview/CHANGELOG.md)
+* [Active Model CHANGELOG](https://github.com/rails/rails/blob/v5.0.0/activemodel/CHANGELOG.md)
+* [Active Record CHANGELOG](https://github.com/rails/rails/blob/v5.0.0/activerecord/CHANGELOG.md)
+* [Active Support CHANGELOG](https://github.com/rails/rails/blob/v5.0.0/activesupport/CHANGELOG.md)
+* [Active Job CHANGELOG](https://github.com/rails/rails/blob/v5.0.0/activejob/CHANGELOG.md)
+* [Railties CHANGELOG](https://github.com/rails/rails/blob/v5.0.0/railties/CHANGELOG.md)
+
+Rails 5.0 also ships with [Turbolinks 5][turbolinks-v5] – the one with native iOS and Android wrapper implementations! If you've dismissed Turbolinks in the past, I urge you to checkout Sam Stephenson's RailsConf presentation: [Turbolinks 5: I Can't Believe It's Not Native!][turbolinks-presentation].
+
+There's even more detail in [the full Rails 5.0 release notes][release-notes] and Claudio did a [nice little slide deck][claudio-slides] walking through some of his favorite improvements (and removals!). And I did a brand-new [let's build a blog in Rails 5][rails5-video] video to demonstrate the basics.
```

Comment:
Should we add in @claudiob's last name? This seems the only instance where a person is mentioned by first name only.